docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
```

Comment:
@aidenkeating There are cases where we point the user to update default vars. See official docs [Section 5.2.2](https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.4/html/installing_rhmap/provisioning-rhmap-core#before-the-installation) so I think it is also fine to point them to the vars folder in this case too.